scripts/src/main/resources/scripts/command/az

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+function doSetup() {
+  if command -v az &> /dev/null
+  then
+    if [ "${1}" != "silent" ] && ! doIsQuiet
+    then
+      doEcho "Azure CLI is already installed at $(command -v az)"
+    fi
+  else
+    doRequireNotBatch
+    if doIsWindows
+    then
+      # Get leatest release
+      if [ -z "${AZURECLI_VERSION}" ]
```

Comment:
I did not notice this before but as the commandlet is called `az` and also the installed command from MS Azure is called `az` we should use `AZ_VERSION` instead of `AZURECLI_VERSION`. The documentation simply tells that the version is configured via `«TOOL»_VERSION` so the user has to guess what `«TOOL»` is. The natural expecation from `az` would be `AZ` rather than `AZURECLI`.